mongo/datadog_checks/mongo/data/conf.yaml.example

```diff
@@ -93,32 +93,64 @@ instances:
     #
     # collections_indexes_stats: false
 
-    ## @param custom_queries - list - optional
+    ## @param custom_queries - list of mappings - optional
     ## Define custom queries to collect custom metrics on your Mongo
     ## See https://docs.datadoghq.com/integrations/guide/mongo-custom-query-collection to learn more.
     #
     # custom_queries:
     #   - metric_prefix: mongo
     #     query: <MONGO_COMMAND_QUERY>
     #     fields:
-    #       - field_name: <FIELD_NAME>
-    #         name: <METRIC_SUFFIX>
-    #         type: <METRIC_TYPE>
+    #     - field_name: <FIELD_NAME>
```

Comment:
We usually indent list in our yamls with two extra spaces. We don't seem to be able to control indentation, no big deal but we could look into that in the future.

Reply:
cc @ofek